### DISCUSSION TITLE
Slider ripple size

### DISCUSSION THREAD

**Discussion by ajaskiewiczpl** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2733] [Q&A] [answered]
How can I adjust size of the ripple "shadow" while dragging? It's too big and doesn't look right in my application. I tried to change `RippleAssist.RippleSizeMultiplier` but it doesn't have effect for the Slider control. Is it possible to change?

![image](https://user-images.githubusercontent.com/10135548/172444705-14d14c62-5383-43b0-ad19-acc85539dc4c.png)

**Comment by ElieTaillard** [comment, marked as answer]
`RippleAssist.RippleSizeMultiplier` let you change the pressed feedback.

![image](https://user-images.githubusercontent.com/54487782/172641398-6c3ecb05-a437-475c-9398-de46242f3fcd.png)

I don't think you could easily change the mouse over size in the library.

However, you can submit a new feature request if you want us to implement this feature:
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/new?assignees=&labels=enhancement%2C+evaluation+required&template=feature_request.md&title=
